### PR TITLE
Feature/11962 home page

### PR DESCRIPTION
**Pull request by erochOnGit**

### CODE REVIEW

File: src/web/theme/ui/molecules/IconWithLabel/IconWithLabel.js [hunk 1]
```diff
@@ -3,18 +3,28 @@ import PropTypes from "prop-types";
...
-  );
+const IconWithLabel = ({ icon, children, type = "up" }) => {
+  if (type === "up") {
+    return (
+      <span className="icon-with-label">
+        <Icon icon={icon} />
+        <span>{children}</span>
+      </span>
+    );
+  } else if (type === "down") {
```
Comment (hihuz) [inline src/web/theme/ui/molecules/IconWithLabel/IconWithLabel.js]: The "else if" statement is unneeded here, if you return nothing the component will throw

File: src/web/theme/modules/ProductList/ProductList.js [hunk 2]
```diff
@@ -1,16 +1,54 @@
...
+  products,
+  currentProductIndex,
+  setCurrentProductIndex
+}) => {
   return (
-    <MediaGrid>
-      {products.map(product => <ProductItem key={product.sku} {...product} />)}
-    </MediaGrid>
+    <div className="product-list">
+      {reassurances.map((reassurance, index) => {
```
Comment (hihuz) [inline src/web/theme/modules/ProductList/ProductList.js]: I don't really like the fact that you have two separate objects representing your product here (reassurance and products[index]).

What you could have done was append the additional data to your products in a withProps HoC, e.g.
```
withProps(props => ({
products: props.products.map((product, index) => ({ ...product, ...reassurances[index] }))
})
```
This way you would only manipulate "products" in the component

Comment (hihuz) [reply src/web/theme/modules/ProductList/ProductList.js]: (Not sure we have the plugin for object spread though in FC Lite :slightly_smiling_face: )

**Review by hihuz — APPROVED** [review]
This looks fine except for a couple of comments I have made

File: src/web/theme/modules/ProductList/ProductHomeView/ProductHomeView.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  product,
+  isCurrent,
+  nextProduct,
+  viewProduct
+}) => {
+  const productHomeViewClasses = `product-home-view--${
+    isCurrent ? "current" : "inactive"
+  }`;
+  return (
+    <div className={productHomeViewClasses}>
```
[the remark this answers is not in the document]
Comment (erochOnGit) [reply src/web/theme/modules/ProductList/ProductHomeView/ProductHomeView.js]: ok

File: src/web/theme/modules/ProductList/ProductHomeView/ProductHomeView.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+      </div>
+      <div className="product-home-view__action">
+        <div className="product-home-view__action__view-product">
+          <Link to={`product/${product.sku}`} type="important">
+            View Product
+          </Link>
+        </div>
+        <div className="product-home-view__action__next-product">
+          <Button
+            onClick={() => {
```
[the remark this answers is not in the document]
Comment (erochOnGit) [reply src/web/theme/modules/ProductList/ProductHomeView/ProductHomeView.js]: oups

File: src/web/theme/modules/ProductList/ProductList.js [hunk 5]
```diff
@@ -1,16 +1,54 @@
...
+        return (
+          <ProductHomeView
+            key={products[index].name + index}
+            accentTitle={reassurance.accentTitle}
+            Title={reassurance.Title}
+            contentProductIntro={reassurance.contentProductIntro}
+            product={products[index]}
+            isCurrent={(index === currentProductIndex) & true}
```
[the remark this answers is not in the document]
Comment (erochOnGit) [reply src/web/theme/modules/ProductList/ProductList.js]: I would like it to be very true ... no i'm kidding ;)

File: src/web/theme/modules/ProductList/ProductList.js (hunk 2 above)
Comment (erochOnGit) [reply src/web/theme/modules/ProductList/ProductList.js]: OOoooooooooo it's look so cool ! I would like to do it !

**Pull request merged**